connector/build.gradle.kts

```diff
@@ -2,9 +2,4 @@
 // Licensed under the MIT license.
 import org.openapitools.generator.gradle.plugin.tasks.GenerateTask
 
-dependencies {
```

Comment:
If we remove this dependencies in projects, will we able to compile them independantly?